workflows/build.yml

```diff
@@ -0,0 +1,46 @@
+name: Build
```

Comment:
☕️💡 this is build yaml for pre flight PR checks when a PR against this repo will be open.